euphonic/util.py

```diff
@@ -68,8 +68,7 @@ def is_gamma(qpt: np.ndarray) -> bool | np.ndarray:
         provided
     """
     tol = 1e-15
-    isgamma = np.sum(np.absolute(qpt - np.rint(qpt)), axis=-1) < tol
-    return isgamma
+    return np.sum(np.absolute(qpt - np.rint(qpt)), axis=-1) < tol
```

Comment:
`np.allclose` ? 

Alternatively [`is_integer`](https://docs.python.org/3/library/stdtypes.html#float.is_integer):
```
return all(map(float.is_integer, qpt))

Reply:
Ah, are you looking for `np.isclose` then? with a reduce in axis 1?

Reply:
`np.all(np.isclose(qpt, np.rint(qpt)), axis=-1)` seems good, do you see any gotchas there?

Reply:
Tolerance (can be passed to `isclose`)? Could also chain ops and avoid an extra wrapping with:
```python
np.isclose(qpt, np.rint(qpt), atol=1e-15).all(axis=-1)
```
if you find that clearer.

I do find this form expresses what you're actually asking better at first glance!

Reply:
Yeah I had the `atol` in code but missed here for some reason. Nice dot-chaining, will put that in.

(I do find it hard to remember in numpy which methods can dot-chain and which work slightly differently in their function form.)